sources/editor/Stride.Assets.Presentation/AssetEditors/ProjectWatcher.cs

```diff
@@ -232,12 +232,15 @@ private async Task<AssemblyChangedEvent> FileChangeTransformation(FileEvent e)
 
                 var needProjectReload = string.Equals(trackedAssembly.Project.FilePath, changedFile, StringComparison.OrdinalIgnoreCase);
 
+                var directoryName = Path.GetDirectoryName(trackedAssembly.Project.FilePath) + "\\";
+                var changedFileDirectoryName = Path.GetDirectoryName(changedFile) + "\\";
```

Comment:
That directory separator would break under linux afaict, use `Path.DirectorySeparatorChar` instead https://learn.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?view=net-8.0

Reply:
allright then i try out ToOsPath as well

Reply:
did it